scripts/combine/printImpacts.py

```diff
@@ -13,9 +13,9 @@ def parseArgs():
     return parser.parse_args()
 
 
-def printImpacts(args,fitresult,POI='Wmass'):
-    impacts,labels,_ = input_tools.readImpacts(fitresult, not args.ungroup, sort=args.sort, POI=POI, normalize = False)
-    unit = 'MeV' if POI=='Wmass' else 'n.u. %'
+def printImpacts(args,fitresult,poi='Wmass'):
```

Comment:
Maybe input_tools being a folder and then having different modules in it would make more sense: utilities.input_tools.combinetf

Reply:
Ok, I have made a separate folder "io_tools" where I put the input and output tools. This creates of course changes in a lot of files but ok